x-pack/test_role.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
**question** is this file being referenced or used anywhere?

Reply:
not being referenced anywhere. removing.